__tests__/unit/Response.test.ts

```diff
@@ -44,18 +36,20 @@ it("Check Response Class without sending the 'entity' field", () => {
   );
 });
 
-it("Check Response Class with successfully correctly usage", () => {
-  const res = new FakeRes();
-
-  const response = new Response(res);
+it("Check Response Class with successfully correctly usage", async () => {
+  const response = new ResponseHandler(res);
 
   const { entities } = response;
 
+  const user = await User.create({
```

Comment:
Use [Faker.js](https://github.com/marak/Faker.js/) to fill in the fields for that object